test/test_distributions.py

```diff
@@ -116,6 +124,9 @@ def _mvn_to_scipy(loc, cov, prec, tril):
     T(dist.MultivariateNormal, np.arange(6, dtype=np.float32).reshape((3, 2)), None, None,
       np.array([[1., 0.], [0., 1.]])),
     T(dist.MultivariateNormal, 0., None, np.broadcast_to(np.identity(3), (2, 3, 3)), None),
+    T(dist.LowRankMultivariateNormal, np.zeros(2), np.array([1, 0]), np.diag(np.array([1, 1]))),
+    T(dist.LowRankMultivariateNormal, np.arange(6, dtype=np.float32).reshape((2, 3)),
+      np.arange(12, dtype=np.float32).reshape((3, 4)), np.diag(np.array([1, 2, 3]))),
```

Comment:
Could you change `cov_factor` to something like `np.arange(6, dtype=np.float32).reshape((3, 2))` so it is indeed "low rank" (though full rank still works, it won't give us the benefit of low-rank algebra tricks). :D